src/context.rs

```diff
@@ -1000,9 +1003,7 @@ impl CDFContext {
       coeff_base_eob_cdf: av1_default_coeff_base_eob_multi,
       coeff_base_cdf: av1_default_coeff_base_multi,
       coeff_br_cdf: av1_default_coeff_lps_multi
-    };
-
-    c
```

Comment:
oh you mean this... by removing the return.

Reply:
Yes, there's no reason to create a new bindings.